packages/specs/schema/src/components/default/nextMapper.ts

```
@@ -0,0 +1,87 @@
+import {JsonSchema} from "../../domain/JsonSchema.js";
+import type {JsonSchemaOptions} from "../../interfaces/JsonSchemaOptions.js";
+import {execMapper, registerJsonSchemaMapper} from "../../registries/JsonSchemaMapperContainer.js";
+import {getGenericsOptions} from "../../utils/generics.js";
+import {mergeSchema} from "../../utils/mergeSchema.js";
+
+function getMapper(input: JsonSchema, mapper: string) {
+  return input && input.isClass ? mapper || "class" : "any";
+}
+
+function buildAndMergeSchemas(mapper: string, schema1: JsonSchema, schema2: JsonSchema, options: JsonSchemaOptions) {
+  let schema = execMapper(getMapper(schema2, mapper), [schema2], {
```

Biome (1.9.4)

[error] This let declares a variable that is only assigned once.

'schema' is never reassigned.

Safe fix: Use const instead.


(lint/style/useConst)